src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/Operators.vb

```diff
@@ -955,13 +949,6 @@ Namespace Microsoft.VisualBasic.CompilerServices
         Public Shared Function NegateObject(ByVal Operand As Object) As Object
 
             Dim tc As TypeCode = GetTypeCode(Operand)
```

Comment:
Dead code, as `GetTypeCode` is implemented as

```vb
Private Shared Function GetTypeCode(ByVal o As Object) As TypeCode
    If o Is Nothing Then
        Return TypeCode.Empty
    Else
        Return o.GetType.GetTypeCode
    End If
End Function
```

so `tc == TypeCode.Empty` if `Operand Is Nothing`